Add finite markov & misc fixes

najuzilu opened the pull request:
* Add new ipynb example
* Fixed reference in `ar1_processes_MyST.ipynb`
* Add new reference file for `inventory_dynam` example
* Removed main reference file
* Update index file to include new ipynb example

mmcky commented:
hey @najuzilu when we merged this in -- did we add `Reference` sections to the notebooks?

mmcky commented:
also what was the reason for having individual reference files? Just to have reference section for each file?

najuzilu commented:
Yes, and I fixed the path to their respective ref.bib files. I added the individual ref files as a quick fix for [this issue](https://github.com/ExecutableBookProject/myst-parser.example-project/issues/11).
